.circleci/config.yml

```diff
@@ -26,6 +26,35 @@ defaults: &defaults
   working_directory: ~/repo
 
 jobs:
+  build_elixir_1_10_otp_23:
+    docker:
+      - image: erlang:23.0
+    environment:
+          ELIXIR_VERSION: 1.10.3-otp-23
+          LC_ALL: C.UTF-8
+    <<: *defaults
+    steps:
+      - checkout
+      - <<: *install_system_deps
+      - <<: *install_elixir
+      - <<: *install_hex_rebar
+      - restore_cache:
+          keys:
+            - v1-mix-cache-{{ checksum "mix.lock" }}
+      - run: mix deps.get
+      - run: mix compile --warnings-as-errors
+      - run: mix test --include inclusion --include integration
+      - run: mix format --check-formatted
+      - run: MIX_ENV=docs mix docs
```

Comment:
I thought `mix docs` had a `preferred_cli_env` set so forcing `MIX_ENV=docs` here shouldn't be necessary. If `hex.build`, `hex.publish`, and `credo` don't have preferred CLI environments, you might want to add them to make them more convenient too.